client/layout/guided-tours/site-title-tour.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	Next,
+	Quit,
+	Continue,
+	Link,
+	ButtonRow,
+} from 'layout/guided-tours/config-elements';
+import {
+	hasSelectedSiteDefaultSiteTitle,
+	isUserOlderThan,
+	isEnabled,
+	canUserEditSettingsOfSelectedSite,
+	isAbTestInVariant,
+} from 'state/ui/guided-tours/contexts';
+import Gridicon from 'components/gridicon';
+import { isDesktop } from 'lib/viewport';
+
+const TWO_DAYS_IN_MILLISECONDS = 2 * 1000 * 3600 * 24;
+
+export const SiteTitleTour = makeTour(
+	<Tour
+		name="siteTitle"
+		version="20161010"
+		path="/stats"
+		when={ and(
+						isEnabled( 'guided-tours/site-title' ),
+						isDesktop,
+						hasSelectedSiteDefaultSiteTitle,
+						canUserEditSettingsOfSelectedSite,
+						isUserOlderThan( TWO_DAYS_IN_MILLISECONDS ),
+						isAbTestInVariant( 'siteTitleTour', 'enabled' )
+					)
+		}>
+		<Step name="init" placement="right" next="click-settings">
+			<p>
+				{
+					translate( "Hey there! We noticed you haven't changed the title of your site yet. Want to change it? " )
```

Comment:
The `design` and `theme` tours are eschewing `translate`ing new strings _for now_, to avoid generating new strings for translators to translate before we're sure we want to keep said strings.